### ISSUE TITLE
Chatbot Arena conversation data

### ISSUE THREAD

**Issue by wlin-at** [OpenGVLab/Multi-Modality-Arena#10]
Hi, 

thanks for the efforts in the great work! 
I would like to ask whether you plan to open-source the Chatbot Arena conversation data. 

Thanks in advance!

Best, Wei

**Comment by vishaal27** [comment]
Yes, I am interested in this too, are there plans to release? Thanks!
